generators/chipyard-helpers/src/main/scala/chipyard-helpers/CodingStyle.scala

```diff
@@ -0,0 +1,53 @@
+class ExampleModule extends Module {
```

Comment:
This ought to just be in the examples, and directly referenced in the docs as a style guide...I don't think there's any value in putting this in a separate package

Reply:
Sure. I just put things here so it was more separated. We can choose where this lives later.